components/core/footer/FooterSocial.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div class="w-full flex justify-center items-center flex-wrap">
+        <ext-link
+            v-for="(item, index) in dataConfig"
+            :key="`landing-footer-icon-${index}`"
```

Comment:
**suggestion:** I guess we also need to define the convention of naming the keys of items in array. e.g. `[page]-[block]-[element]-[index]` or something like that.

Reply:
This is the part I missing in last meeting. There are so many kinds of naming convention in the repository. I totally agree that we should define one of our naming convention here.